.github/workflows/destroy.yml

```diff
@@ -16,6 +16,27 @@ jobs:
         with:
           terraform_version: 0.14.0
 
+      - name: Clear S3 bucket
+        env:
+          CF_USERNAME: ${{ secrets.GOVPAAS_DEV_USERNAME }}
+          CF_PASSWORD: ${{ secrets.GOVPAAS_DEV_PASSWORD }}
+        run: |
+          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+          echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+          sudo apt-get update
+          sudo apt-get install cf7-cli
+          cf api https://api.london.cloud.service.gov.uk
+          cf auth
+          cf target -o "dfe" -s "earlycareers-framework-dev"
+          cf create-service-key ecf-csv-review-pr-${{ github.event.number }} ecf-csv-review-pr-${{ github.event.number }}-key -c '{"allow_external_access": true}'
+          CREDS=$(cf service-key ecf-csv-review-pr-${{ github.event.number }} ecf-csv-review-pr-${{ github.event.number }}-key)
+          AWS_ACCESS_KEY_ID=$(echo $CREDS | tail -7 | jq -r '.aws_access_key_id')
```

Comment:
For my understanding's sake, what's `tail -7` and `jq -r` doing here exactly?

Reply:
service-key gives
```
{
 "aws_access_key_id": "xxx",
 "aws_region": "eu-west-2",
 "aws_secret_access_key": "xxx",
 "bucket_name": "xxx",
 "deploy_env": ""
}
```
(with the blank line at the start)
`tail` takes the last 7 lines here, without the blank line so that `jq` doesn't complain
`jq` is for json, and pulls out the values. We need `-r` for raw, so that the values aren't wrapped in quotes